login/login.c

```diff
@@ -243,33 +243,45 @@ void login_syslog(glome_login_config_t* config, pam_handle_t* pamh,
   }
 }
 
-// read_stdin reads printable characters from stdin into buf. It returns:
+// read_stdin reads characters from stdin into buf. It returns:
```

Comment:
Could you please update the docstring to say that we're always advancing stdin to a newline or EOF?

Reply:
Done.